lib/features/Auth/login/data/datasources/shared_prefs.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:shared_preferences/shared_preferences.dart';
+
+// This class contains the constants - Email id Key and Password key
+class Constant{
+final String key_Email="email";
+final String key_Password="password";
+}
+
+// This class contains the code for saving and retrieval of email and password of current user
+class Preferences{
+  Preferences(){}
+// To set the Emailid of the current logging in user
+  Future<bool> set_Email(String email) async{
```

Comment:
Name non-constant identifiers using lowerCamelCase
```suggestion
  Future<bool> setEmail(String email) async {
```